src/touchid/TouchID.mm

```diff
@@ -0,0 +1,212 @@
+#define SECURITY_ATTRIBUTE_ACCOUNT CFSTR("KeepassXC TouchID Keys")
+#define TOUCH_ID_PROMPT_MESSAGE CFSTR("authenticate to access KeepassXC database!")
```

Comment:
From the screenshot, looks like the touchid UI automatically adds a _._ to the end of this message, making the exclamation mark seem out of place.

Reply:
You're right, I'll remove it and replace the whole thing with QCoreApplication::translate(...).